packages/cli-core/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "@aws-amplify/backend-function": "^0.8.0-beta.1",
```

Comment:
cli-core must not depend on backend-function. We could either
1. hardcode the path in the core package with a comment explaining the coupling to backend-functions or
2. Put the paths in platform-core and import in both backend-function and cli-core

Reply:
I am leaning towards 1. @sobolk and @Amplifiyer also had strong opinions on adding verticals stuff in `platform-core` [here](https://github.com/aws-amplify/amplify-backend/pull/984/#discussion_r1480341390)

Reply:
yes.